src/modules/CourseThumbnailAPI/_lib/options.ts

```diff
@@ -0,0 +1,32 @@
+import chrome from 'chrome-aws-lambda'
+
+const exePath =
```

Comment:
Is this mean we need Chrome to dev this project?

Reply:
Only if you want to dev the thumbnail part